extensions/vertx/deployment/pom.xml

```diff
@@ -34,7 +34,7 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-vertx</artifactId>
         </dependency>
-	<dependency>
+	    <dependency>
```

Comment:
Doubt this passes the automated formatter…

Reply:
Hum, I don't have this issue.
I had to fix it because it was using a tab instead of spaces.

Reply:
Well, if you ran it, and it's fine…

Reply:
The automated formatter doesn't check XML files, only Java.

But, yes, there is still a tab there.